changelog/2766.doc.2.rst

```diff
@@ -0,0 +1 @@
+Renamed :file:`docs/_cff_to_rst.py` to :file:`docs/_author_list_from_cff.py`.
```

Comment:
🏷️ Mentioning in the name that this is for creating an author list should lead to more readable code.